src/typing_extensions.py

```diff
@@ -3533,7 +3533,14 @@ def __getitem__(self, parameters):
                 )
                 for item in parameters
             ]
-            return typing._GenericAlias(self, tuple(parameters))
+            if sys.version_info >= (3, 10):
+                return typing.GenericAlias(self, tuple(parameters))
```

Comment:
It is re-exported in the `typing` module, but that's really an accident. You're meant to import it from the `types` module :)

```suggestion
                return _types.GenericAlias(self, tuple(parameters))
```